frontend/deb/distro/worker.go

```diff
@@ -79,6 +79,7 @@ func (cfg *Config) Worker(sOpt dalec.SourceOpts, opts ...llb.ConstraintsOpt) (ll
 		Run(
 			dalec.WithConstraints(opts...),
 			AptInstall(cfg.BuilderPackages, opts...),
+			dalec.WithMountedAptCache(cfg.AptCachePrefix),
```

Comment:
Was this just not using the shared apt cache before?

Reply:
Yes, somehow got left out when I moved things around.